src/pages/Profile.js

```diff
@@ -26,27 +29,40 @@ function ProfilePage() {
         }
     };
 
+    const handleDeleteUser = async (e) => {
+        e.preventDefault();
+        try {
+            await axios.get('http://localhost:8005/deleteUser', { params: { accessToken: accessToken } });
+            localStorage.removeItem('accessToken');
+            navigate('/');
+        } catch (error) {
+            navigate('/');
+        }
+    };
+
     useEffect(() => {
-        async function getUser() {
+        async function getUserProperties(username) {
             try {
-                let response = await axios.get('http://localhost:8005/user', { params: { accessToken: accessToken } });
-                setUserData(response.data);
+                let response = await axios.get(`http://localhost:8002/fetchPropertiesByUser`, { params: { userId: username /*accessToken: accessToken*/ } });
```

Comment:
This endpoint does not exist in the inventory service, besides we should use the service manager for backend requests

Reply:
yes, waiting for the others to finish their implementations of the endpoints